packages.dhall

```diff
@@ -1,6 +1,4 @@
 let upstream =
-      https://raw.githubusercontent.com/purescript/package-sets/psc-0.13.0-20190614/src/packages.dhall sha256:5cbf2418298e7de762401c5719c6eb18eda4c67ba512b3f076b50a793a7fc482
+      https://github.com/purescript/package-sets/releases/download/psc-0.13.8-20210226/packages.dhall sha256:7e973070e323137f27e12af93bc2c2f600d53ce4ae73bb51f34eb7d7ce0a43ea
```

Comment:
also note that dhall has made the previous package set expression unusable. if i didn't publish the normalized forms back then i would have had to do this manually yet again to try to make this able to be consumed by newest dhall. worth remembering that this will be a source of breakages also in the future.